jwst/datamodels/photom.py

```diff
@@ -74,69 +59,58 @@ class NrcWfssPhotomModel(ReferenceFileModel):
         of instrument mode parameters and photometric conversion
         factors associated with those modes.
 
-        - filter: str[12]
-        - pupil: str[15]
-        - order: int16
-        - photmjsr: float32
-        - uncertainty: float32
-        - nelem: int16
-        - wavelength: float32[*]
-        - relresponse: float32[*]
-        - reluncertainty: float32[*]
+       - filter: str[12]
+       - subarray: str[15]
+       - photmjsr: float32
+       - uncertainty: float32
+       - nelem: int16
+       - wavelength: float32[*]
+       - relresponse: float32[*]
+       - reluncertainty: float32[*]
 
     """
-    schema_url = "nrcwfss_photom.schema"
+    schema_url = "mirlrs_photom.schema"
 
 
-class NirissPhotomModel(PhotomModel):
+class MirMrsPhotomModel(ReferenceFileModel):
     """
-    A data model for NIRISS photom reference files.
+    A data model for MIRI MRS photom reference files.
 
     Parameters
-    __________
-    phot_table : numpy table
-        Photometric flux conversion factors table
-        A table-like object containing row selection criteria made up
-        of instrument mode parameters and photometric conversion
-        factors associated with those modes.
+    ----------
+    init : any
+        Any of the initializers supported by `~jwst.datamodels.DataModel`.
 
-        - filter: str[12]
-        - pupil: str[12]
-        - order: int16
-        - photmjsr: float32
-        - uncertainty: float32
-        - nelem: int16
-        - wavelength: float32[5000]
-        - relresponse: float32[5000]
+    data : numpy array
+        An array-like object containing the pixel-by-pixel conversion values
+        in units of DN / sec / mJy / pixel.
```

Comment:
This needs parentheses.  Also, shouldn't it be (MJy / pixel) / (DN / s)?

Reply:
Yes, I missed this one. Good eye!